apps/erudio/frontend/src/lib/session.ts

```diff
@@ -0,0 +1,5 @@
+import rspc from "../api-setup";
+
+const createSession = (login = false) => rspc.createQuery(() => ['user.me'], { retry: login ? false : 3, refetchOnWindowFocus: !login })
```

Comment:
I this this is logical error
condition ? exprIfTrue : exprIfFalse

Reply:
No. On login page we're hitting the endpoint once just to see if we have a session and quickly show the login form if we don't. On (auth) pages we may want to retry, especially if it's just a cache miss and the user might be in the middle of something.

Reply:
Ok the parameter name is slightly misleading the retry could be better but it fine

Reply:
Also cache miss doesn't return failure it just forwards the request to actual check so it may take up to around 30-40ms longer but if user is logged in it won't fail